Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -341,13 +457,19 @@ public override void RunSetUpdate(ObjectId runSetLiteDbId, ObjectId runnerLiteDb
                 runner.Elapsed = gingerRunner.BusinessFlows[0].Elapsed;
             }
             runner.RunStatus = (liteDbBFList.Count > 0) ? liteDbBFList[0].RunStatus : eRunStatus.Automated.ToString();
+
+            SetRunnerChildCounts(runner);
```

Comment:
why we need to calc the Runner inside RunSetUpdate? wasn't it already calculated in SetReportRunner()?